docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -236,3 +238,100 @@ This would make it more uniform with the existing `${component.<key>}` variables
 +     brokers: ${config.kafka_brokers}
 +     schemaRegistryUrl: ${config.schema_registry.url}
 ```
+
+## Summary
+
+!!! warning
+
+    [**Helm will not find your (long) old release names anymore.**](#use-hash-and-trim-long-helm-release-names-instead-of-only-trimming)
+
+??? example "defaults.yaml"
+
+    ```diff
```

Comment:
Just double-check if it is correctly rendered on the `dev` branch of the docs.
the .md file looks a bit strange
<img width="336" alt="image" src="https://github.com/bakdata/kpops/assets/20357405/fd9165d1-f610-4a97-9cf7-95914cf09fea">

Reply:
nvm looks great
<img width="738" alt="image" src="https://github.com/bakdata/kpops/assets/20357405/ab51b7eb-e99e-4811-8d76-92031b370d7e">